docs/howto/installing-local-extensions.md

```diff
@@ -5,34 +5,96 @@ level: intermediate
-extensions site.
+Bolt supports extensions that are not included on Bolt's extensions site at
+[market.bolt.cm][market]. Instead, these extensions are part of the "custom
+code" for your project.
+
+The major benefit of this is that you can easily "hack away" on an extension
+where it's not relevant if it can be distributed later on. For example, a quick
+proof-of-concept, or the addition of a simple Twig tag for a specific purpose.
+This is especially true where the extension is application specific and doesn't
+need to be shared across multiple Bolt sites.
 
 There are a couple of caveats:
 
-- Updates are **not** available though the web UI
+- Updates are **not** available though the web UI in Bolt's backend. This is
+  simply because there's no link between your "local" extension, and wherever
+  it might originate from.
 - You are responsible for managing the location of assets, including specifying
-the location of supporting files and copying/storing web assets in the public
-folder
+  the location of supporting files and copying/storing web assets from the
+  extensons `web/` folder to the public folder, like `public_html/`
 
-**Note:** If your local extension requires libraries from Packagist, simply add
-them to the `composer.json` in the root directory of your Bolt install by
-performing a `composer require author/package`.
+Apart from that, the local extensions support all functionality that's found in
+the "Real" extensions.
 
-Option 1 - Extensions in your base application
-------
+There are two ways to ensure a local extension gets picked up by the Bolt
+installation:
 
-If you are using the composer install method for Bolt whereby Bolt is installed
-as a dependency of your main application then it makes sense for some
-extensions to be maintained as part of your application's source code.
+ - [Extensions in your extensions/local folder][local-folder]
+ - [Extensions in your base application][base-app]
 
-This is especially true where the extension is application specific and doesn't
-need to be shared across multiple Bolt sites.
+Option 1 - Extensions in your extensions/local folder
+-----------------------------------------------------
 
-For the extension to load the class must be autoloadable from the root composer.json
-if you already have a PSR-4 definition in your composer.json then you can add your
-extension to the same path, if not then you can add one that may look something
-like this:
+If you place a valid Bolt extension inside the `extensions/local` directory
+then the class will be accessible to your Bolt app.
+
+For example, if the local folder does not exist, then create it and inside
+`extensions/local` make an additional folder, `[Extensionname]` and inside that folder add a valid Bolt extension class and save as `[Extensionname]Extension.php`.
+
+Your extensions folder would look something like this:
+
+```
+extensions
+├── local/
+│   └── Listing/
+│   │   ├── ListingExtension.php
```

Comment:
I am still wondering if we shouldn't keep the documented approach in line with how we do Composer based extensions, as I know for fact we get people installing things like BoltForms, or forks of certain extensions as locals. That way the documented migration path is nothing more than the autoloader settings in `composer.json`